src/content/homepage/charts/second-chart/convert-msaxlsx-to-html.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+if [ $# -ne 1 ]; then
+     echo "usage: $0 <xlsx-filename>"
+     exit 1
+fi
+
+# https://github.com/dilshod/xlsx2csv (this gracefully handles unicode nastiness, thankfully).
+pip install xlsx2csv
+xlsx2csv -d '|' $1 tmp-msa.csv
+
+echo '<select name="hmda_chart_2_msa" id="hmda_chart_2_msa">
+<option selected value="CBSA00000">U.S. Total</option>' > msa-dropdown.html
```

Comment:
I made this a variable but not a command-line option. I hesitate on making it totally configurable, because the user would then need to decide on a select id/name.